pom.xml

```diff
@@ -79,6 +79,7 @@
         <module>tests</module>
         <module>udf-compiler</module>
         <module>udf-examples</module>
+        <module>ml</module>
```

Comment:
The build instructions imply this isn't building by default, but it looks like it is.  Should this be built only with a specific profile?

Reply:
@jlowe  Thank you Jason for the review and suggestions, I've kept all those comments and will try to fix/answer them in another new repo. According offline discussion, we will request a new repo and put those work there. 
Thanks again!